pkg/util/mapper/mapper.go

```diff
@@ -373,6 +383,27 @@ func (rom *readOnlyMapper) GetBool(key string, defaultValue bool) bool {
 	return b
 }
 
+// GetList returns a string list which contains the value set by key split using the
+// provided delimiter with each entry trimmed of space. If the value doesn't exist,
+// nil is returned
+func (rom *readOnlyMapper) GetList(key string, delimiter string) []string {
+	value := rom.Get(key, "")
+	if value == "" {
+		return nil
+	}
+
+	split := strings.Split(value, delimiter)
+
+	// reuse slice created for split
```

Comment:
Fancy

Reply:
Just trying to avoid the `strings.Split` allocation churn our key tuples were generating months ago -- it can be somewhat disorienting to read at first, but in general, we avoid allocating a new backing storage every time we create a slice.